interpreter/src/main/scala/pl/touk/nussknacker/engine/marshall/ScenarioParser.scala

```diff
@@ -0,0 +1,19 @@
+package pl.touk.nussknacker.engine.marshall
+
+import cats.data.Validated.{Invalid, Valid}
+import pl.touk.nussknacker.engine.canonicalgraph.CanonicalProcess
+import pl.touk.nussknacker.engine.canonize.ProcessCanonizer
+import pl.touk.nussknacker.engine.graph.EspProcess
+
+object ScenarioParser {
```

Comment:
Thanks for extracting it +1 Can u replace other places of this code duplication?

Reply:
done in https://github.com/TouK/nussknacker/pull/2633/commits/65e1f25b0cf1bbc6cfea9cd4c26375075c11d50b